src/cmake/__init__.py

```diff
@@ -3,10 +3,14 @@
 import subprocess
 import sys
 
-from ._version import get_versions
+from ._version import version as __version__
+
+__all__ = ["__version__", "CMAKE_DATA", "CMAKE_BIN_DIR", "CMAKE_DOC_DIR", "CMAKE_SHARE_DIR", "cmake", "cpack", "ctest"]
```

Comment:
Should we really expose all those CMAKE variables ?

Reply:
I know CMAKE_BIN_DIR is used other places. I don’t remember where.